etl/steps/data/garden/wb/2023-07-10/education.py

```diff
@@ -0,0 +1,153 @@
+        tb.rename(columns={column: new_column_name}, inplace=True)
+        tb[new_column_name].metadata.description = " ".join(
+            [
+                description + "." "World Bank variable id: " + indicator_to_find + ".",
+                source,  # Change this when the new metadata format is ready
+            ]
+        )
+        tb[new_column_name].metadata.title = name
+        tb[new_column_name].metadata.display = {}
+
+        #
+        # Update metadata units, short_units and number of decimal places to display depending on what keywords the variable name contains
+        #
+
+        def update_metadata(table, column, display_decimals, unit, short_unit=" "):
+            """
+            Update metadata attributes of a specified column in the given table.
+
+            Args:
+            table (obj): The table object containing the column.
+            column (str): Name of the column whose metadata is to be updated.
+            display_decimals (int): Number of decimal places to display.
+            unit (str): The full name of the unit of measurement for the column data.
+            short_unit (str, optional): The abbreviated form of the unit. Defaults to an empty space.
+
+            Returns:
+            None: The function updates the table in-place.
+            """
+            table[column].metadata.display["numDecimalPlaces"] = display_decimals
+            table[column].metadata.unit = unit
+            table[column].metadata.short_unit = short_unit
+
+        # Convert the 'name' variable to lowercase for easier text matching.
+        name_lower = name.lower()
+
+        # Define a list of keywords associated with percentages.
+        percentage_unit = ["%", "percentage", "share of", "rate"]
+
+        # Check if any keyword from the percentage_unit list is present in 'name_lower' and ensure "number" is not in 'name_lower'.
+        if any(keyword in name_lower for keyword in percentage_unit) and not ("number" in name_lower):
+            update_metadata(tb, new_column_name, 1, "%", "%")
+        elif "ratio" in name_lower:
+            update_metadata(tb, new_column_name, 1, "ratio", " ")
+        elif "number of pupils" in name_lower:
+            update_metadata(tb, new_column_name, 0, "pupils", " ")
+        # Check if the column name contains "number", but not "rate" or "pasec".
+        elif "number" in name_lower and not ("rate" in name_lower) and not ("pasec" in name_lower):
+            update_metadata(tb, new_column_name, 0, "people", " ")
+        elif "(years)" in name_lower or "years" in name_lower:
+            update_metadata(tb, new_column_name, 1, "years", " ")
+        elif "index" in name_lower:
+            update_metadata(tb, new_column_name, 1, "index", " ")
+        # Check for the presence of currency-related keywords in 'name_lower'.
+        elif "USD" in name_lower or "$":
+            update_metadata(tb, new_column_name, 1, "USD", "$")
```

Comment:
Please check that currency indicators follow [the FASTT harmonization guidelines](https://docs.google.com/document/d/1tcknTryJAxOR0yv8x38s3jkONq-TY3T4HNCEcZQ_c-s/edit).